test/web-platform-tests/to-run.yaml

```diff
@@ -824,3 +824,15 @@ failure.html: [fail, Fetch is not implemented]
 toascii.window.html: [fail, Unknown]
 url-constructor.html: [fail, url.search vs. url.searchParams propagation]
 urlencoded-parser.html: [fail, Fetch is not implemented]
+
+---
+
+DIR: websockets
+
+Create-Secure-extensions-empty.htm: [fail, Buggy test as the test does not take into account the mandatory permessage-deflate extension]
```

Comment:
All browsers fail this test, with Chrome and Firefox failing the same way we do: https://wpt.fyi/websockets/Create-Secure-extensions-empty.htm